### PR TITLE
Fix Code format (including comment)

### PR DESCRIPTION
**Pull request by fengshunli**
Signed-off-by: fengshunli <1171313930@qq.com>

### Which issue(s) this PR fixes:

Fixes #

### Does this PR introduced a user-facing change?

```release-note

```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by wenchajun — APPROVED** [review]

File: pkg/fluentd/router/pluginstore.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ type LabelRouter struct {
 	Routes []*Route `json:"routes,omitempty"`
 }
 
-// Each fluentd config instance will create a route pluginstore.
+// NewRoutePlugin Each fluentd config instance will create a route pluginstore.
```
Comment (benjaminhuo) [inline pkg/fluentd/router/pluginstore.go:33]:
```suggestion
// Each fluentd config instance will create a route pluginstore.
```

File: pkg/fluentd/router/pluginstore.go [hunk 2]
```diff
@@ -83,7 +83,7 @@ func (r *Route) NewRoutePlugin() (*params.PluginStore, error) {
 	return ps, nil
 }
 
-// The global router to store routes
+// NewGlobalRouter The global router to store routes
```
Comment (benjaminhuo) [inline pkg/fluentd/router/pluginstore.go:86]:
```suggestion
// The global router to store routes
```

File: pkg/operator/collector-statefulset.go [hunk 3]
```diff
@@ -11,7 +11,7 @@ import (
 )
 
 var (
-	// Buffer path for single process
+	// DefaultBufferPath Buffer path for single process
```
Comment (benjaminhuo) [inline pkg/operator/collector-statefulset.go:14]:
```suggestion
	// Buffer path for single process
```

Comment (benjaminhuo) [reply pkg/operator/collector-statefulset.go:14]: Thanks for the PR, just wondering why adding function names like NewRoutePlugin, NewGlobalRouter, and DefaultBufferPath to the comments?

**Review by benjaminhuo — APPROVED** [review]

File: pkg/fluentd/router/pluginstore.go (hunk 1 above)
Comment (fengshunli) [reply pkg/fluentd/router/pluginstore.go:33]: Add variable descriptions before comments, which is a standard format requirement

File: pkg/operator/collector-statefulset.go (hunk 3 above)
Comment (fengshunli) [reply pkg/operator/collector-statefulset.go:14]: Add variable descriptions before comments, which is a standard format requirement

Comment (wanjunlei) [reply pkg/operator/collector-statefulset.go:14]: Using the function or parameter name as the beginning of the comment is correct, but we need to ensure that the comment is grammatical. Perhaps. 

`DefaultBufferPath Buffer path for single process` is not a good comment.
`DefaultBufferPath defines the buffer path for single process` is better.

I think these comments all need to be adjusted.

Comment (fengshunli) [reply pkg/operator/collector-statefulset.go]: updated

File: pkg/fluentd/router/pluginstore.go (hunk 1 above)
Comment (benjaminhuo) [reply pkg/fluentd/router/pluginstore.go:33]: No need to add function name like NewRoutePlugin and NewGlobalRouter to the function's comments

Comment (fengshunli) [reply pkg/fluentd/router/pluginstore.go:33]: Using the function or parameter name as the beginning of the comment is correct, but we need to ensure that the comment is grammatical. .

Comment (benjaminhuo) [reply pkg/fluentd/router/pluginstore.go:33]: Seems that both comments are not grammatical:
```
NewRoutePlugin Each fluentd config instance will create a route pluginstore.
NewGlobalRouter The global router to store routes Member
```

Comment (fengshunli) [reply pkg/fluentd/router/pluginstore.go:33]: image

Comment (benjaminhuo) [reply pkg/fluentd/router/pluginstore.go:33]: > NewRoutePlugin Each fluentd config instance will create a route pluginstore.
> NewGlobalRouter The global router to store routes Member

I mean we cannot simply append a function name to the original comments to conform with the guideline, we need to make the entire comment conform the grammar maybe like this:

```shell
NewRoutePlugin will create a route pluginstore for each fluentd config instance.
NewGlobalRouter will create a global router to store routes Member
```

Comment (fengshunli) [reply pkg/fluentd/router/pluginstore.go]: updated

**Review by benjaminhuo — APPROVED** [review]

**Comment by benjaminhuo** [comment]
Thanks for the contribution!

**Pull request merged**